frontend/src/modules/addonpromotion/components/AddonPromotion.css

```diff
@@ -0,0 +1,50 @@
+.addon-promotion + header {
+    margin-top: 44px;
+}
+
+.addon-promotion {
+    background: #272a2f;
+    border-bottom: 1px solid #333941;
+    height: 44px;
+    position: fixed;
+    top: 0;
+    width: 100%;
+    z-index: 20;
+}
+
+.addon-promotion .container {
+    align-items: center;
+    background: #ff3366dd;
```

Comment:
This color (associated with error state) might be a littler too much drawing attention. But it's probably best from the state palette in terms of contrast.

Reply:
Yup, this is already down tuned a little bit from the early feedback. I'll make it slightly more transparent, i.e. dark.